src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -212,7 +213,7 @@ const DiscoverySearch = ({
               radius: 35
             }}>
             <View style={styles.wrapperDeleteIcon}>
-              <IconClear width={9} height={10} iconColor={COLORS.black} />
+              <IconClear width={9} height={10} fill={COLORS.gray400} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `IconClear` component has been changed from black to gray400. Ensure that this change doesn't affect the visibility of the icon, especially if the background is also a similar color.
```diff
- <IconClear width={9} height={10} iconColor={COLORS.black} />
+ <IconClear width={9} height={10} fill={COLORS.gray400} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->